docs/backends/amazon-S3.rst

```diff
@@ -17,11 +17,11 @@ in version 1.2
 Settings
 --------
 
-To use boto3 set::
+To use boto3, ensure that the Pypi packages ``boto`` and ``boto3`` are installed and set::
```

Comment:
Nope, I just tried it, it needs `boto` installed, as well as `boto3`. Try it yourself.